### PR TITLE
UDISE+ School Details Scraper

### PR DESCRIPTION
**Pull request by devesh-002**
Pushing my scraper in a new branch.

### CODE REVIEW

**Comment by vijaynyaya** [comment]
There are numerous issues with the current scarper. Taking inspiration from how we extracted the Input Survey, @SaileshReddyM can you please review the scraper in `get_raw_data.py`.

**Comment by vijaynyaya** [comment]
There are numerous issues with the current scarper. Taking inspiration from how we extracted the Input Survey, @SaileshReddyM can you please review the scraper in `get_raw_data.py`.

File: projects/udise/scraper_devesh.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+from selenium.webdriver.support.ui import Select
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.remote.command import Command
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import base64
+
+sleep_time=0.2
+
+try:
```
Comment (SaileshReddyM) [inline projects/udise/scraper_devesh.py]: Avoid using "os" for path. Instead "Path" is preferred "from pathlib import Path".  Example: 
   raw_datadir = Path(__file__).parents[2] / "data" / "raw"
   download_dir = raw_datadir / "downloads"

File: projects/udise/scraper_devesh.py [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+except:
+    pass
+try:
+     open("error.txt", 'w').close()
+except:
+    pass
+
+url="https://src.udiseplus.gov.in/locateSchool/schoolSearch"
+# facilities and room details
+def main_func():
```
Comment (SaileshReddyM) [inline projects/udise/scraper_devesh.py]: Avoid writing everything under one main function. Split the main function into "Pure Functions". You can have a function that solely makes a driver and returns it. You can use Pure functions for all repetitive codes.

**Review by vijaynyaya — APPROVED** [review]

**Pull request merged**